### ISSUE TITLE
Constant runtime related to traitor panel and changelings

### ISSUE THREAD

**Issue by paprka** [tgstation/-tg-station#7663]
Steps to reproduce:

Make someone a changeling using the traitor panel. Then click this:
![clip 2015-02-11 at 12 40 16](https://cloud.githubusercontent.com/assets/7247232/6156019/5559139c-b1eb-11e4-8d05-c67315e1fb5b.png)

```
runtime error: Cannot read null.mutations
proc name: Life (/mob/living/carbon/human/Life)
  source file: life.dm,65
  usr: null
  src: Esteban Power (/mob/living/carbon/human)
  call stack:
Esteban Power (/mob/living/carbon/human): Life(2)
Mobs (/datum/subsystem/mobs): fire()
/datum/controller/game_control... (/datum/controller/game_controller): process()
```

This runtime spams forever.

**Comment by Razharas** [comment]
DNA transfers are fucked in multiple places around the code
In fact im surprised lings work at all

**Comment by phil235** [comment]
Can't reproduce, very likely fixed by #10923 which changed how changeling handle their absorbed DNAs.
